### PR TITLE
Release notes for OpenNMS Horizon 31.0.9

### PR DESCRIPTION
**Pull request by jeffgdotorg**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/projects/NMS/versions/10970/tab/release-report-all-issues

### CODE REVIEW

**Review by mershad-manesh — APPROVED** [review]

File: docs/modules/releasenotes/pages/changelog.adoc [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+
+==== Release 31.0.9
+
+Release 31.0.9 contains one CVE-related security fix, a generous helping of other bug fixes, and a several small enhancements intended to improve supportability.
+
+The codename for Horizon 31.0.9 is https://wikipedia.org/wiki/$$Ballokume$$[_Ballokume_].
+
+===== Breaking changes
+
+* This release has moved to a newer major version of Spring Security to address a number of CVEs, which necessitated changes to the `$OPENNMS_HOME/jetty-webapps/opennms/WEB-INF/applicationContext-spring-security.xml` file, so if you have modified this file in your installs, be sure to note your changes so you can re-apply them to the updated version.
```
Comment (Bonrob2) [inline docs/modules/releasenotes/pages/changelog.adoc:14]:
```suggestion
* This release has moved to a newer major version of Spring Security to address a number of CVEs, which necessitated changes to the `$OPENNMS_HOME/jetty-webapps/opennms/WEB-INF/applicationContext-spring-security.xml` file, so if you have modified this file in your installs, be sure to note your changes so you can reapply them to the updated version.
```

File: docs/modules/releasenotes/pages/changelog.adoc [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+* Simplify the installation docs (Issue https://issues.opennms.org/browse/NMS-15518[NMS-15518])
+* Docs: Add info about XSLT to XmlCollector (Issue https://issues.opennms.org/browse/NMS-15693[NMS-15693])
+* Doc: Update DNS provisioning import adapter docs (Issue https://issues.opennms.org/browse/NMS-15694[NMS-15694])
+
+===== Task
+
+* Remove unsupported configuration from documentation on Cortex Plugin (Issue https://issues.opennms.org/browse/NMS-14969[NMS-14969])
+* Multiple CVEs for Axis 1.4 (Issue https://issues.opennms.org/browse/NMS-15061[NMS-15061])
+
+===== Bug
```
Comment (Bonrob2) [inline docs/modules/releasenotes/pages/changelog.adoc:33]:
```suggestion
=== Bugs
```

File: docs/modules/releasenotes/pages/changelog.adoc [hunk 3]
```diff
@@ -1,6 +1,52 @@
...
+* Fixing typo for event uei.opennms.org/internal/schedOutagesChanged (Issue https://issues.opennms.org/browse/NMS-15421[NMS-15421])
+* Sentinels need local copy of thresholding config. (Issue https://issues.opennms.org/browse/NMS-15422[NMS-15422])
+* Event Datetime element parsing changed between M2018 and M2021 (Issue https://issues.opennms.org/browse/NMS-15471[NMS-15471])
+* install script checks for equality of myuser and RUNAS before sourcing opennms.conf (Issue https://issues.opennms.org/browse/NMS-15610[NMS-15610])
+* timeout is using DEFAULT_TIMEOUT value instead of the value from properties file when no -t option is specified (Issue https://issues.opennms.org/browse/NMS-15664[NMS-15664])
+* Stop packaging activemq-web-console.war (Issue https://issues.opennms.org/browse/NMS-15686[NMS-15686])
+* Database deadlock caused by JdbcFilterDao (Issue https://issues.opennms.org/browse/NMS-15696[NMS-15696])
+* Karaf SSH locks up if connections are terminated improperly (Issue https://issues.opennms.org/browse/NMS-15714[NMS-15714])
+
+===== Story
```
Comment (Bonrob2) [inline docs/modules/releasenotes/pages/changelog.adoc:44]:
```suggestion
=== Story
```

**Comment by Bonrob2** [comment]
@jeffgdotorg , there are several issues with the headings that need to be fixed here. I apologize that I didn't review this sooner, but I've been in meetings since 1:00 pm.

**Pull request merged**